openmls/src/framing/message_in.rs

```diff
@@ -243,13 +248,14 @@ impl From<PublicMessageIn> for ProtocolMessage {
     }
 }
 
-#[cfg(any(feature = "test-utils", test))]
-impl From<MlsMessageIn> for ProtocolMessage {
-    fn from(msg: MlsMessageIn) -> Self {
+impl TryFrom<MlsMessageIn> for ProtocolMessage {
```

Comment:
If we change this implementation and make it available for non-test purposes, wouldn't it make sense to change the `impl Into` input in `fn process_message` to `impl TryFrom`? I tried it and it's a bit annoying because we need to put constraints on the Error type in TryFrom, but I think it makes for a nicer API.